tiledb/vfs.py

```diff
@@ -1,7 +1,7 @@
 import io
 import os
 from types import TracebackType
-from typing import List, Optional, Type, Union
+from typing import ByteString, List, Optional, Type, Union
```

Comment:
@kounelisagis `ByteString` is deprecated. What would you think about using `bytes | bytearray | memoryview` as suggested at https://docs.python.org/3/library/typing.html#typing.ByteString ?